src/tests/Common/Coreclr.TestWrapper/CoreclrTestWrapperLib.cs

```diff
@@ -219,8 +223,9 @@ static bool CollectCrashDump(Process process, string path)
             else if (OperatingSystem.IsLinux() || OperatingSystem.IsMacOS())
             {
                 createdump.StartInfo.FileName = "sudo";
-                createdump.StartInfo.Arguments = $"{createdumpPath} " + arguments;
+                createdump.StartInfo.Arguments = $"{createdumpPath} --crashreport " + arguments;
                 createdump.StartInfo.EnvironmentVariables.Add("DOTNET_DbgEnableElfDumpOnMacOS", "1");
```

Comment:
Remove this, should no longer be necessary.

Reply:
Why is that?

Reply:
`DOTNET_DbgEnableElfDumpOnMacOS` was a .NET 5 shortcut to generate dumps. Now there's macOS dumps in the native format.